### PR TITLE
Feature 17 upload images component

### PR DESCRIPTION
**Pull request by IgorPopach**

### CODE REVIEW

File: client/src/components/ImageUploader/ImageUploader.js [hunk 1]
```diff
@@ -156,8 +156,17 @@ class ImageUploader extends Component {
...
+          </div>
       </div>;
+    
+    const renderNotMember = 
+      <div className="upload-card">
+        <p>Only members can upload images!</p>
+    </div>;
 
     const uploadCard = 
       <div className="upload-card" onClick={(event) => event.stopPropagation()} >
```
Comment (yodeco) [inline client/src/components/ImageUploader/ImageUploader.js]: As Sasha told, (event) => better event =>
Optional

File: client/src/components/ImageUploader/DropZone/DropZone.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    constructor(props) {
+        super(props);
+        this.state = {
+            hightlight: false,
+        };
+        this.fileInputRef = React.createRef();
+    }
+
+    openFileDialog = (event) => {
+        console.log('openFileDialog', this.props.disabled)
```
Comment (yodeco) [inline client/src/components/ImageUploader/DropZone/DropZone.js]: () can be removed

File: client/src/components/ImageUploader/DropZone/DropZone.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    }
+
+    fileListToArray = (list) => {
+        const array = [];
+        for (var i = 0; i < list.length; i++) {
+            array.push(list.item(i));
+        }
+        return array;
+    }
+    componentDidMount() {
```
Comment (yodeco) [inline client/src/components/ImageUploader/DropZone/DropZone.js]: maybe componentDidMount can be removed?

File: client/src/components/ImageUploader/DropZone/DropZone.scss [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+    50% {
+      -webkit-transform: rotate(180deg);
+      transform: rotate(180deg);
+    }
+    100% {
+      -webkit-transform: rotate(360deg);
+      transform: rotate(360deg);
+    }
+  }
+  @-webkit-keyframes lds-eclipse {
```
Comment (yodeco) [inline client/src/components/ImageUploader/DropZone/DropZone.scss]: Why 2 times with vendor prefix and without? Webpack should have postcss/autoprefixer, so with that we are ok with vendor prefixes. Could you recheck that pls? Or maybe I miss some point? If plugin is enabled, remove pls with vendor prefix

**Review by yodeco — APPROVED** [review]

**Review by YuliaFursovych — APPROVED** [review]

**Pull request merged**